snippets/snippets.json

```diff
@@ -1,146 +1,158 @@
 {
```

Comment:
about 130 changes in lines are due to changing the indentation. If the indent changes are needed shouldn't it be in a separate feature branch?

Reply:
I looked at some other `.json` files in here and they all used 2 spaces, so I thought it's the default in this repo. I'm sorry if I guessed wrong, I could change it back.

Reply:
Yeah, I added prettier to this project and probably forgot this file. LGTM! Tyvm